app/middlewares/auth.js

```diff
@@ -0,0 +1,23 @@
+const { badRequestError } = require('../errors');
+const logger = require('../logger');
+const { verifyToken } = require('../utils/jwt');
+
+exports.authenticated = (req, res, next) => {
+  try {
+    const accessToken = req.headers.authorization;
+
+    if (!accessToken) throw new Error();
+
+    const payload = verifyToken(accessToken);
+
+    const { user: id, name } = payload;
+    if (!id || !name) throw new Error();
```

Comment:
I do not think this is necessary, nor throw the error, you could simply add to the req what the `verifyToken` returns

Reply:
Santi, I check that because sometimes I have worked with id token and refresh token, so I check the structure to as not to confuse them